### ISSUE TITLE
add_pie() - svg pielayer not positioned correctly.

### ISSUE THREAD

**Issue by flo12392** [ropensci/plotly#1002]
---
Since the last plotly update, the svg layer of the pie chart is not positioned properly. whereas previously the pielayer was at least as big as the svg main layer, there is now always a blank space at the top. This makes the pie much smaller than it could be. See the attached image for an example (added axis to improve the example).

[http://oi65.tinypic.com/2ylpes4.jpg](http://oi65.tinypic.com/2ylpes4.jpg)
I am running plotly 4.6.0 and R 3.3.3. Reverting to plotly 4.5.6 solves the issue. I tried to find other causes for this but I didn't find any, I hope I am not overlooking anything. This minimal example gives me the same issue:

```r
require(dplyr)
require(plotly)
df <-data.frame(id=c(1,2),values=c(0.6,0.4))
p <-plot_ly() %>%
  add_pie(df, 
          values = df$values)
```

**Comment by cpsievert** [comment]
This sounds like something that may have to be resolved at the plotly.js level. They prefer examples via codepen -- https://codepen.io/cpsievert/pen/XRqjzX

> the svg layer of the pie chart is not positioned properly

Could you please clarify what you mean and post [here](https://github.com/plotly/plotly.js)? Thanks!

**Comment by flo12392** [comment]
Thanks for creating the codepen, I have no prior experience in JavaScript so that would have been difficult for me. I have added the issue to plotly.js, also clarified with a screenshot from the result of your codepen: [https://github.com/plotly/plotly.js/issues/1679](https://github.com/plotly/plotly.js/issues/1679)

**Comment by flo12392** [comment]
@cpsievert I tried to revert the plotly.js version to 1.7.0 in the codepen and it yields the same results. However, look at the differences in the plot under plotly 4.5.6 and 4.6.0:
[Google Drive link to images](https://drive.google.com/open?id=0ByGbN6t9KhP_bjUwb1Y5VXJtdEk)
The plot under 4.6.0. does not use all the available plotting space, whereas in 4.5.6 it did.
